runtime/admin/src/test/java/org/apache/polaris/admintool/relational/jdbc/RelationalJdbcAdminProfile.java

```diff
@@ -42,8 +40,6 @@ public Map<String, String> getConfigOverrides() {
   @Override
   public List<TestResourceEntry> testResources() {
     return List.of(
-        new TestResourceEntry(
-            PostgresRelationalJdbcLifeCycleManagement.class,
-            Map.of(INIT_SCRIPT, "org/apache/polaris/admintool/init.sql")));
```

Comment:
turns out the init script (which creates 2 more databases) isnt needed at all by current tests